web-api-db-ctrl/elasticsearch/mapping.json

```diff
@@ -14,6 +14,9 @@
             "tags": {
                 "type": "keyword"
             },
+            "parent": {
+                "type": "keyword"
+            },
```

Comment:
:wrench:  I will change this as part of some out of band review feedback.

Reply:
I decided against it  the degree of complexity this adds to the code in ways which are prone to have bugs later down the line makes it not worth it to do so AFIK.

The reason this is a field instead of just using the part in the id of the document is so that it's indexed and we can delete by-parent id.

This is needed due upserts no longer working with split documents, as the number of childs in a upsert can have changed.

But at the point of time we do the upserts we don't always know which children the document had _before_ we started to reingest it.  So we have the choice between
either doing additiona PG queries and then sending a long list to the server, or just have a by-parent index and use delete by query in ES which is easier to write, less complex and probably more performant/cheap, too.